Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -363,12 +363,14 @@ public class Fuzzer {
         } else {
             logger.warning("Execution for type collection did not succeeded, outcome: \(execution.outcome)")
         }
+        // Save result of runtime types collection to Program
+        program.typeCollectionStatus = TypeCollectionStatus(typeCollectionOutcome: execution.outcome)
```

Comment:
I think either the guard in line 347 should set the CollectionStatus to `.notAttempted` or, probably preferably, there should be a `precondition(program.typeCollectionStatus == .notAttempted)` before the guard to make sure this function is used properly and the status is always consistent